src/indra_cogex/client/queries.py

```diff
@@ -784,7 +794,7 @@ def get_evidences_for_stmt_hashes(
         A mapping of stmt hash to a list of evidence objects for the given
         statement hashes.
     """
-    stmt_hashes_str = ",".join(f'"{h}"' for h in stmt_hashes)
+    stmt_hashes_str = ",".join(f"{h}" for h in stmt_hashes)
```

Comment:
just `str(h)`?

Reply:
I guess, it's the same number of characters :)